keps/sig-node/127-user-namespaces/README.md

```diff
@@ -464,18 +463,15 @@ components that implement the interface.
 
 [Pod Security Standards](https://k8s.io/docs/concepts/security/pod-security-standards)
 define three different policies to broadly cover the whole security spectrum of
-Kubernetes, while the User Namespaces feature should integrate into them. This
-will happen only if the feature is graduated to GA, which _may_ result in
-changing the `Restricted` profile to disallow host user namespaces for stateless
-Pods.
+Kubernetes, while the User Namespaces feature should integrate into them.
 
 The Pod Security will relax in a controlled way for pods which enable user
 namespaces. This behavior can be controlled by an API Server Feature Gate, which
 allows an early opt-in for end users. The overall burden to ensure that all
-nodes will honor user namespaces is on the cluster admin, though. The relaxation
-in detail means, that if user namespaces are enabled, then the following fields
-won't be restricted any more because they always have to refer to the user
-inside the container:
+nodes will honor user namespaces is on the cluster admin, though.
+
+The relaxation in detail means, that if user namespaces are enabled, then the following fields
+will be accessible to pods in a restricted namespace:
 
 - `spec.securityContext.runAsNonRoot`
```

Comment:
are you saying that when userns is enabled 

runAsNonRoot can be set to false  and runAsUser/runAsGroup can be set to 0? 

If so then lets just say it like that? WDYT?

Reply:
thanks I reworded to be a bit less verbose, PTAL @vinayakankugoyal